python/cuml/test/test_random_forest.py

```diff
@@ -646,3 +646,25 @@ def test_rf_memory_leakage(fil_sparse_format, column_info, nrows):
         handle.sync()
         delta_mem = free_mem - cuda.current_context().get_memory_info()[0]
         assert delta_mem == 0
+
+
+@pytest.mark.parametrize('max_features', [1.0, 'auto', 'log2', 'sqrt'])
+@pytest.mark.parametrize('max_depth', [10, 13, 16])
+@pytest.mark.parametrize('n_estimators', [10, 20, 100])
+@pytest.mark.parametrize('n_bins', [8, 9, 10])
+def test_create_classification_model(max_features,
```

Comment:
it would be nice to have a test which would refit the same model over and over again, this would test the updates to fit function.